media-tv/tvbrowser-bin/metadata.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="project">
+    <email>java@gentoo.org</email>
```

Comment:
Have you asked if java team wants to maintain this? Add the proxy-maint project otherwise. Nevertheless, person should become before project.